### DISCUSSION TITLE
Increase engagement/feedback from (academic) users

### DISCUSSION THREAD

**Discussion by EwoutH** [mesa/mesa#2287] [Future of Mesa]
Currently, engagement levels and especially direct feedback on features from users is really low. Especially feedback from academic users seems lacking. We have over a thousand academic users who spend significant time on Mesa yearly, but less than a single percent has any kind of engagement with us. This seems to be the trend among all layers of users, from bachelor students to PhDs to professors.

This engagement and feedback crucial for a few things:
- Feedback on what's currently lacking helps us to develop new features and enhancements
- Feedback can help us prioritize what to spend our limited time on
- Feedback on new and especially experimental features can help iterate on them before we stabilize them
- Engagement can lead to new developers, new ideas and on the very long term more maintainers for Mesa

I know for a fact there are opinions and ideas about Mesa functionality, documentation and all kind of other things. So how do we get them reported to us?

**Comment by wang-boyu** [comment]
MASON (a java abm framework from GMU) had a NSF-funded workshop to collect feedbacks from users: https://www.nsf.gov/awardsearch/showAward?AWD_ID=1205626. It then had another NSF grant for further development based on these feedbacks: https://www.nsf.gov/awardsearch/showAward?AWD_ID=1727303.

A main challenge for us is that these grants are awarded to universities. Might still worth considering? @tpike3 @jackiekazil

**Reply by jackiekazil** [reply]
I have been chatting with a few folks at NSF and other places. there are other options without university association -- will update at the next meeting. These things are often slow moving though -- or perfect timing.

**Reply by EwoutH** [reply]
Sounds like an interesting option!

We do have university association right? It was a George Mason project, TU Delft has many users and Is actively involved in development.
